### PR TITLE
✨Source Yahoo finance: Make Connector Compatible with Builder

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
* Make the connector compatible with the builder

## How
* Import schema to manifest
* Update docs, 
* Completely made compatible with the builder

## User Impact
* No impact, chore for maintainability.

## Can this PR be safely reverted and rolled back?
- [x] YES 💚

### CODE REVIEW

**Comment by btkcodedev** [comment]
CI green ✅
Removed Parameters macro ✅

Ready to merge ✅
image

File: airbyte-integrations/connectors/source-yahoo-finance-price/source_yahoo_finance_price/manifest.yaml [hunk 1]
```diff
@@ -1,312 +1,73 @@
-version: 0.50.0
+version: 0.79.1
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-yahoo-finance-price/source_yahoo_finance_price/manifest.yaml:1]: Weird that this didn't get bumped to newer version

File: airbyte-integrations/connectors/source-yahoo-finance-price/source_yahoo_finance_price/manifest.yaml [hunk 2]
```diff
@@ -1,312 +1,73 @@
...
+
 type: DeclarativeSource
+
 check:
   type: CheckStream
   stream_names:
     - price
-streams:
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-yahoo-finance-price/source_yahoo_finance_price/manifest.yaml:7]: That's also interesting. Don't get me wrong, this should work, but Builder is supposed to ref out schemas, and here it inlined it.

File: airbyte-integrations/connectors/source-yahoo-finance-price/source_yahoo_finance_price/manifest.yaml (hunk 1 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-yahoo-finance-price/source_yahoo_finance_price/manifest.yaml:1]: I pruned Docker volume two days before, isn't this the latest version?

**Review by natikgadzhi — APPROVED** [review]

File: docs/integrations/sources/yahoo-finance-price.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ The Airbyte Source for [Yahoo Finance Price](https://finance.yahoo.com/)
 
 | Version | Date       | Pull Request                                             | Subject                                                                         |
 | :------ | :--------- | :------------------------------------------------------- | :------------------------------------------------------------------------------ |
+| 0.2.5   | 2024-05-25 | [38602](https://github.com/airbytehq/airbyte/pull/38602) | Remove parameters macro and make compatible with builder                        |
```
Comment (girarda) [inline docs/integrations/sources/yahoo-finance-price.md:9]:
```suggestion
| 0.2.5   | 2024-05-28 | [38602](https://github.com/airbytehq/airbyte/pull/38602) | Remove parameters macro and make compatible with builder                        |
```

**Review by girarda — APPROVED** [review]

**Pull request merged**